Ginger/Ginger/ValueExpression/ValueExpressionEditorPage.xaml.cs

```diff
@@ -168,9 +167,11 @@ private void BuildItemsTree()
 
         private void AddGlobalParameters()
         {
+            TreeViewItem Parent = AddOrGetCategory("Data");
+            //TreeViewItem child = AddOrGetSubCategory("Models Global Parameters", Parent);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6356765)